hoomd/md/EvaluatorPairGauss.h

```diff
@@ -25,6 +25,50 @@
 #define DEVICE
 #endif
 
+struct gauss_params
```

Comment:
I'm not sure if this fits with the design of the rest of the API or is easily compatible with pybind11, but could you define these parameter structs within the evaluator so that you don't need to worry about polluting the namespace with parameter names (that will all look like `<potential>_params` anyway)? Something like:

```c++
class EvaluatorPairGauss
{
    typedef struct gauss_params
    {
    Scalar epsilon;
    Scalar sigma;
    } param_type;
}

EvaluatorPairGauss::param_type params(eps,sigma);
```

Reply:
This change seems to be compatible with pybind11, and I agree that it cleans things up, so I will make this change